### PR TITLE
Reserve enums for QNX platform

### PR DESCRIPTION
**Pull request by stonesthrow**
Reserve for new extensions QNX platform and QNX native buffer

### CODE REVIEW

**Comment by oddhack** [comment]
LGTM. Please mark Approved to Merge when ready (or just merge it yourself).

**Comment by stonesthrow** [comment]
@oddhack , Jon, ready to merge. We have QNX/Blackberry teams approval to move forward. Please merge this enum allocation.

File: api/egl.xml [hunk 1]
```diff
@@ -1078,6 +1078,10 @@
             <unused start="0x3540" end="0x354F"/>
     </enums>
 
+    <enums namespace="EGL" start="0x3550" end="0x355F" vendor="QNX" comment="Reserved for Mike Gorchak of QNX (QNX platform and native buffer)">
+            <unused start="0x3550" end="0x355F"/>
+    </enums>
+
 <!-- Please remember that new enumerant allocations must be obtained by
      request to the Khronos API registrar (see comments at the top of this
      file) File requests in the Khronos Bugzilla, EGL project, Registry
```
Comment (oddhack) [inline api/egl.xml:1087]: I should really edit this :-)

**Comment by stonesthrow** [comment]
> > @oddhack , Jon, ready to merge. We have QNX/Blackberry team's approval to move forward. Please merge this enum allocation.
> 
> Apparently it can't be merged in Draft state and despite having admin rights on the repo, I can't seem to un-Draft it - can you do that? Then you can just click the Merge button if you want, else I'll do it later today.

Yeah, I didn't see how to do that. I'll have to figure it out.

File: api/egl.xml (hunk 1 above)
Comment (stonesthrow) [reply api/egl.xml:1087]: I thought you wanted requesters to prepare the change. Sorry.

Comment (oddhack) [reply api/egl.xml:1087]: I just meant that we haven't used Bugzilla for half a decade or more.

**Comment by stonesthrow** [comment]
> > > @oddhack , Jon, ready to merge. We have QNX/Blackberry team's approval to move forward. Please merge this enum allocation.
> > 
> > 
> > Apparently it can't be merged in Draft state and despite having admin rights on the repo, I can't seem to un-Draft it - can you do that? Then you can just click the Merge button if you want, else I'll do it later today.
> 
> Yeah, I didn't see how to do that. I'll have to figure it out.

I thought draft would keep the updates and review private until ready, thus no email noise until ready to review and merge. But I guess not.

**Comment by stonesthrow** [comment]
@oddhack - Ready to merge. Thanks

**Pull request merged**